docs/internal-documentation/maintenance/ReleaseProcess.md

```diff
@@ -73,6 +73,10 @@ Generate Makefile, META.yml and others.
 > **Note:** Ignore the warning from the above command about the missing
 > META.yml. The META.yml is created by the same command at a later stage.
 
+> **Note:** For Zonemaster-Engine ignore the warning from the above command
+> about the missing .mo files. These files are created when running `make all`
+> below.
+
 > **Note:** For Zonemaster-LDNS ignore the above command may generate warnings
 > for lots of missing ldns source files.
 > The ldns sources are fetched by the same command at a later stage.
```

Comment:
That is too much text. All the details do not need that to be there. It stops the flow when using the text. Details could be added in bottom of the file as a footnote, if needed. I suggest that three sections are merged into one:

> **Note:** 
> Ignore the warning about the missing META.yml (created at a later stage).
>
> For Zonemaster-Engine ignore the warning about the missing .mo files (created in next
> command).
>
> For Zonemaster-LDNS ignore any generate warnings for lots of missing ldns source files 
> (fetched at a later stage)

Reply:
I implemented your general idea. Please review again.